web/src/components/JsonLdEvent.tsx

```diff
@@ -38,8 +38,8 @@ export const createJsonLd = (event: EventModel, formatter: DateFormatter): WithC
     },
   }
 
-  if (date.endDate.isValid()) {
-    jsonLd.endDate = date.allDay ? date.endDate.format('YYYY-MM-DD') : date.endDate.format()
+  if (date.endDate.isValid) {
+    jsonLd.endDate = date.allDay ? date.endDate.toUTC().toFormat('ff') : date.endDate.toUTC().toISO() ?? ''
```

Comment:
The date should not be converted to `utc`, see https://developers.google.com/search/docs/appearance/structured-data/event#best-practices.
Instead of `ff`, `toISODate` should be used:
```suggestion
    jsonLd.endDate = date.allDay ? date.endDate.toISODate() : date.endDate.toISO() ?? ''
```

Reply:
@steffenkleinle actually testing gives of the right result after including `toUTC()`. Plain `toISO()` or `toISODate()` gives off **not recommended practice** from the link you shared